### PR TITLE
fix: Update copyright holder's name to `EddieHub`

### PR DESCRIPTION
**Pull request by Panquesito7**
#### What does this PR do?

- Update copyright holder's name to `EddieHub`

#### Description of the task to be completed?

None.

#### How can this be manually tested?

I don't think this applies here.

#### Any background context you want to provide?

No.

#### Is there any relevant issue to this PR?

As far as I know, nope.

#### Questions

None.

### CODE REVIEW

**Review by krishdevdb — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Eddie Jaoude Community
+Copyright (c) 2020-2021 EddieHub
```
Comment (FayasNoushad) [inline LICENSE:4]: Why `2020-2021`? I think `2021` is better.

Comment (Panquesito7) [reply LICENSE:4]: The project was created in 2020 (I guess), and it still continues in 2021. 🙂

Comment (Panquesito7) [reply LICENSE:4]: It still continues in 2021, that's why `2020-2021`.

Comment (FayasNoushad) [reply LICENSE:4]: Aa, No problem but its needed to edit in all years.

**Review by shreyaansjain06 — APPROVED** [review]

**Comment by schmelto** [comment]
As mentioned in another PR I`m not sure if EddieHub is a legal entity we can use here @eddiejaoude could you clarify this?

**Comment by eddiejaoude** [comment]
EddieHub is not a legal entity, but I think it would be better than what there is now. We should probably update and then look into this further. Thank you all for updating and checking 👍

**Review by schmelto — APPROVED** [review]

**Pull request merged**